Missing Definition for some jsx-a11y rules

I have a meteor (1.3.2.4) project with react where I added eslint-plugin-shopify. 

All my files complain of missing jsx-a11y rules, on the first line (import React from 'react')
```
  1:1  error    Definition for rule 'jsx-a11y/img-uses-alt' was not found                    jsx-a11y/img-uses-alt
  1:1  error    Definition for rule 'jsx-a11y/label-uses-for' was not found                  jsx-a11y/label-uses-for
  1:1  error    Definition for rule 'jsx-a11y/mouse-events-map-to-key-events' was not found  jsx-a11y/mouse-events-map-to-key-events
  1:1  error    Definition for rule 'jsx-a11y/no-hash-href' was not found                    jsx-a11y/no-hash-href
  1:1  warning  Definition for rule 'jsx-a11y/redundant-alt' was not found                   jsx-a11y/redundant-alt
  1:1  error    Definition for rule 'jsx-a11y/valid-aria-role' was not found                 jsx-a11y/valid-aria-role

```

My package.json is:
```
{
  "name": "board",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "lint": "eslint . --max-warnings 0"
  },
  "author": "",
  "license": "ISC",
  "dependencies": {
    "moment": "^2.13.0",
    "moment-timezone": "^0.5.3",
    "react": "^0.14.8",
    "react-dom": "^0.14.8",
    "react-redux": "^4.4.1",
    "react-router": "^2.0.1",
    "redux": "^3.5.2",
    "redux-thunk": "^2.0.1"
  },
  "eslintConfig": {
    "extends": [
      "plugin:shopify/react"
    ],
    "env": {
      "meteor": true,
      "node": true
    },
    "rules": {
      "no-warning-comments": "off",
      "no-console": "off",
    },
    "plugins": [
      "jsx-a11y"
    ]
  },
  "devDependencies": {
    "eslint": "^2.9.0",
    "eslint-plugin-babel": "^3.2.0",
    "eslint-plugin-import": "^1.6.1",
    "eslint-plugin-jsx-a11y": "^1.0.4",
    "eslint-plugin-shopify": "^11.1.2",
    "eslint-plugin-sort-class-members": "^1.0.1"
  }
}
```

So far, I have removed this lines on "rules" manually now, but I guess there is a problem with the way the shopify plugin uses jsx-a11y.

```
    "rules": {
      "no-warning-comments": "off",
      "no-console": "off",
      "jsx-a11y/img-uses-alt": "off",
      "jsx-a11y/label-uses-for": "off",
      "jsx-a11y/mouse-events-map-to-key-events": "off",
      "jsx-a11y/no-hash-href": "off",
      "jsx-a11y/redundant-alt": "off",
      "jsx-a11y/valid-aria-role": "off"
    },
```

--- a/packages/eslint-plugin-shopify/lib/config/rules/jsx-a11y.js
+++ b/packages/eslint-plugin-shopify/lib/config/rules/jsx-a11y.js
@@ -1,22 +1,36 @@
 // see https://github.com/evcohen/eslint-plugin-jsx-a11y
 
 module.exports = {
+  // Enforce all aria-* props are valid.
+  'jsx-a11y/aria-props': 'error',
+  // Enforce ARIA state and property values are valid.
+  'jsx-a11y/aria-proptypes': 'error',
+  // Enforce that elements with ARIA roles must use a valid, non-abstract ARIA role.
+  'jsx-a11y/aria-role': 'error',
+  // Enforce that elements that do not support ARIA roles, states, and properties do not have those attributes.
+  'jsx-a11y/aria-unsupported-elements': 'error',
+  // Enforce an anchor element's href prop value is not just #.
+  'jsx-a11y/href-no-hash': 'error',
   // Enforce that <img> JSX elements use the alt prop.
-  'jsx-a11y/img-uses-alt': 'error',
+  'jsx-a11y/img-has-alt': 'error',
+  // Enforce <img> alt prop does not contain the word "image", "picture", or "photo".
+  'jsx-a11y/img-redundant-alt': 'warn',
   // Enforce that <label> elements have the htmlFor prop.
-  'jsx-a11y/label-uses-for': 'error',
+  'jsx-a11y/label-has-for': 'error',
   // Enforce that onMouseOver/onMouseOut are accompanied by onFocus/onBlur for keyboard-only users.
-  'jsx-a11y/mouse-events-map-to-key-events': 'error',
+  'jsx-a11y/mouse-events-have-key-events': 'error',
   // Enforce that the accessKey prop is not used on any element to avoid complications with keyboard commands used by a screenreader.
   'jsx-a11y/no-access-key': 'error',
-  // Enforce an anchor element's href prop value is not just #.
-  'jsx-a11y/no-hash-href': 'error',
-  // Enforce that non-interactive, visible elements (such as <div>) that have click handlers use the role attribute.
-  'jsx-a11y/onclick-uses-role': 'off',
-  // Enforce <img> alt prop does not contain the word "image", "picture", or "photo".
-  'jsx-a11y/redundant-alt': 'warn',
   // Enforce that onBlur is used instead of onChange.
-  'jsx-a11y/use-onblur-not-onchange': 'off',
-  // Enforce that elements with ARIA roles must use a valid, non-abstract ARIA role.
-  'jsx-a11y/valid-aria-role': 'error',
+  'jsx-a11y/no-onchange': 'off',
+  // Enforce that elements with onClick handlers must be focusable.
+  'jsx-a11y/onclick-has-focus': 'error',
+  // Enforce that non-interactive, visible elements (such as <div>) that have click handlers use the role attribute.
+  'jsx-a11y/onclick-has-role': 'off',
+  // Enforce that elements with ARIA roles must have all required attributes for that role.
+  'jsx-a11y/role-has-required-aria-props': 'error',
+  // Enforce that elements with explicit or implicit roles defined contain only aria-* properties supported by that role.
+  'jsx-a11y/role-supports-aria-props': 'error',
+  // Enforce tabIndex value is not greater than zero.
+  'jsx-a11y/tabindex-no-positive': 'error',
 };

--- a/packages/eslint-plugin-shopify/package.json
+++ b/packages/eslint-plugin-shopify/package.json
@@ -37,7 +37,7 @@
     "eslint-plugin-chai-expect": "^1.1.1",
     "eslint-plugin-flowtype": "^2.2.6",
     "eslint-plugin-import": "^1.5.0",
-    "eslint-plugin-jsx-a11y": "^0.6.2",
+    "eslint-plugin-jsx-a11y": "^1.0.4",
     "eslint-plugin-lodash": "^1.6.9",
     "eslint-plugin-mocha": "^2.2.0",
     "eslint-plugin-node": "^1.0.0",

